docs/spec/staking/transactions.md

```diff
@@ -28,74 +28,74 @@ A validator is created using the `TxCreateValidator` transaction.
 
 ```golang
 type TxCreateValidator struct {
-	OwnerAddr           sdk.Address
+	Operator            sdk.Address
```

Comment:
formatting is off

Reply:
Strings are same length:

```
alessio@bizet:~$ [ `echo "OwnerAddr           sdk.Address" | wc -m` = `echo "Operator            sdk.Address" | wc -m` ] && echo OK || echo KO
OK
```

Perhaps I misunderstood though.